packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_configuration_netboot_title": "Changer le netboot",
```

Comment:
I am not sure, but i think we can write `Netboot` instead of `netboot`

Reply:
The message is as per the text I have received.